apps/arweave/include/ar_config.hrl

```diff
@@ -198,6 +200,7 @@
 	pool_server_address = not_set,
 	pool_api_key = not_set,
 	pool_worker_name = not_set,
+	peers_list_filter = ?DEFAULT_PEERS_LIST_FILTER,
```

Comment:
I was thinking that this config would actually be a way to set the number of days that would be filtered whenever using the `current` peers list. Alternatively it could be a way to set the days of filtering for each specific peers list (e.g. it could be a multi-use flag that lest you do something like `peers_list_filter lifetime:365 peers_list_filter current:30` so you could set different thresholds or different lists)

As implemented it looks like it only applies to the `/peers` endpoint? I'm not sure that that's that useful since it won't let us configure the peers list actually used by this node when syncing chunks and/or blockchain data.

I suggest we either:
1. remove the option and just use `DEFAULT_PEERS_LIST_FILTER` everywhere. And in that case I'd suggest renaming it to something like `CURRENT_PEERS_LIST_FILTER` to make clear that it only applies to the `current` list and not the `lifetime` or `connected` lists
2. or change the flag so that it lets you configure the filter value for any of the lists (e.g. the multi-use flag example above)


I'm fine with either approach.